src/test/java/org/folio/rest/api/RequestsApiTest.java

```diff
@@ -1521,6 +1522,584 @@ public void canDeleteARequest()
       getResponse.getStatusCode(), is(HttpURLConnection.HTTP_NOT_FOUND));
   }
 
+  @Test
+  public void canExpireASingleOpenUnfilledRequest()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+    UUID id = UUID.randomUUID();
+    UUID itemId = UUID.randomUUID();
+
+    JsonObject requestRequest = new RequestRequestBuilder()
+      .hold()
+      .withId(id)
+      .withRequestExpiration(new DateTime(2017, 7, 30, 10, 22, 54, DateTimeZone.UTC))
+      .withItemId(itemId)
+      .withPosition(1)
+      .withStatus(OPEN_NOT_YET_FILLED)
+      .create();
+    CompletableFuture<JsonResponse> createCompleted = new CompletableFuture<>();
+    client.put(requestStorageUrl(String.format("/%s", id)),
+      requestRequest, StorageTestSuite.TENANT_ID,
+      ResponseHandler.json(createCompleted));
+
+    JsonResponse response = createCompleted.get(5, TimeUnit.SECONDS);
+    assertThat(String.format("Failed to create request: %s", response.getBody()),
+      response.getStatusCode(), is(HttpURLConnection.HTTP_NO_CONTENT));
+
+    CompletableFuture<Void> getExpirationCF = new CompletableFuture<>();
+    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx(), StorageTestSuite.getVertx().getOrCreateContext()).setHandler(res -> {
+      getExpirationCF.complete(null);
+    });
+    getExpirationCF.get(5, TimeUnit.SECONDS);
```

Comment:
I think the code for triggering expiration can be extracted to reduce duplication.

Something like
```suggestion
    expireRequests();

  private void expireRequests() throws InterruptedException, ExecutionException, TimeoutException {
    CompletableFuture<Void> expirationCompleted = new CompletableFuture<>();

    ExpirationTool.doRequestExpiration(StorageTestSuite.getVertx(),
      StorageTestSuite.getVertx().getOrCreateContext())
      .setHandler(res -> expirationCompleted.complete(null));

    expirationCompleted.get(15, TimeUnit.SECONDS);
  }
```

Reply:
moved to separate method, thank you